net-nntp/sabnzbd/sabnzbd-4.1.0.ebuild

```diff
@@ -0,0 +1,168 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_REQ_USE="sqlite"
+
+inherit optfeature python-single-r1 systemd
+
+MY_PV="${PV/_alpha/Alpha}"
+MY_PV="${MY_PV/_beta/Beta}"
+MY_PV="${MY_PV/_rc/RC}"
+
+MY_P="${PN/sab/SAB}-${MY_PV}"
```

Comment:
Just inline the package name here, substitution is longer than doing that, it's unreadable and it's just asking for trouble.